features/gebruik_in_lopende_tekst.feature

```diff
@@ -34,138 +34,126 @@ Rule: gebruikInLopendeTekst voor een persoon zonder adellijke titel of predicaat
+      | voornamen (02.10)                    | Jo Rene    |
+      | adellijke titel of predicaat (02.20) |            |
+      | voorvoegsel (02.30)                  |            |
+      | geslachtsnaam (02.40)                | Groenen    |
+      | aanduiding naamgebruik (61.10)       | E          |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                            |
+      | type                | RaadpleegMetBurgerservicenummer   |
+      | burgerservicenummer | 000000139                         |
+      | fields              | adressering.gebruikInLopendeTekst |
+      Dan heeft de response een persoon met de volgende 'adressering' gegevens
+      | naam                  | waarde                  |
+      | gebruikInLopendeTekst | <gebruikInLopendeTekst> |
+
+      Voorbeelden:
+      | voorbeeld | geslacht | gebruikInLopendeTekst |
+      | man       | M        | de heer Groenen       |
+      | vrouw     | V        | mevrouw Groenen       |
+      | onbekend  | O        | J.R. Groenen          |
 
   Abstract Scenario: gebruikInLopendeTekst van een persoon die nooit huwelijk of partnerschap heeft gehad met aanduidingNaamgebruik "<aanduidingNaamgebruik>"
-    Gegeven het systeem heeft een persoon met de volgende gegevens
-    | naam                        | waarde    |
-    | burgerservicenummer         | 999992934 |
-    | geslachtsaanduiding (04.10) | V         |
-    En de persoon heeft de volgende 'naam' gegevens
-    | naam                                 | waarde |
-    | adellijke titel of predicaat (02.20) |        |
-    | voorvoegsel (02.30)                  | in het |
-    | geslachtsnaam (02.40)                | Veld   |
-    | aanduiding naamgebruik (61.10)       | E      |
-    En de persoon heeft nooit een actueel of ontbonden huwelijk of partnerschap gehad
-    Als personen wordt gezocht met de volgende parameters
-    | naam                | waarde                            |
-    | type                | RaadpleegMetBurgerservicenummer   |
-    | burgerservicenummer | 999992934                         |
-    | fields              | adressering.gebruikInLopendeTekst |
-    Dan heeft de response een persoon met de volgende 'adressering' gegevens
-    | naam                  | waarde              |
-    | gebruikInLopendeTekst | mevrouw In het Veld |
-
-    Voorbeelden:
-    | aanduidingNaamgebruik |
-    | E                     |
-    | P                     |
-    | V                     |
-    | N                     |
+    Gegeven de persoon met burgerservicenummer '000000140' heeft de volgende gegevens
+      | naam                                 | waarde |
+      | geslachtsaanduiding (04.10)          | V      |
+      | adellijke titel of predicaat (02.20) |        |
+      | voorvoegsel (02.30)                  | in het |
+      | geslachtsnaam (02.40)                | Veld   |
+      | aanduiding naamgebruik (61.10)       | E      |
```

Comment:
Wijzig in:
`| aanduiding naamgebruik (61.10) | <aanduidingNaamgebruik> |`

Reply:
Waarom ? 
Als ik dat doe gaat het insert statement mis of moet ik een nieuwe column mapping maken. Ik snap je niet.

Reply:
Als ik de rule in regel 34 goed interpreteer moet de proxy volgens mij voor 'naamgebruik', ongeacht de daadwerkelijke waarde van 'aanduiding naamgebruik (61.10)', de waarde 'E' hanteren als een persoon nooit een actuele of ontbonden huwelijken/partnerschappen heeft gehad. De proxy moet, als 'naamgebruik' een andere waarde dan 'E' heeft, dus checken of dat terecht is. Is er inderdaad geen actuele of ontbonden huwelijk/partnerschap?

Er moet hier volgens mij getest worden of de proxy dat goed doet.
Daarvoor moet de proxy natuurlijk wel de middelen aangereikt krijgen van de GBA API. Vandaar dat ik daarover een opmerking maak in de eerste bullet van [deze opmerking](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/pull/1356/files#r1005540965).

Reply:
OK, nu begrijp ik je en daar ben ik het ook mee eens. Ik las je wijzigingvoorstel verkeerd.